scilpy/image/labels.py

```diff
@@ -67,6 +68,48 @@ def get_binary_mask_from_labels(atlas, label_list):
     return mask
 
 
+def get_labels_from_mask(mask_data, labels=None, background_label=0):
+    """
+    Get labels from a binary mask which contains multiple blobs.
+
+    Parameters
+    ----------
+    mask_data: np.ndarray
+        The mask data.
+    labels: list, optional
```

Comment:
Maybe say the default behavior? Labels will go from 1 to N (number of blobs) if none are given.